operations/model.go

```diff
@@ -76,15 +76,17 @@ func isValidPath(path string) bool {
 // located at ~/.evergreen.yml
 // If you change the JSON tags, you must also change an anonymous struct in hostinit/setup.go
 type ClientSettings struct {
-	APIServerHost      string              `json:"api_server_host" yaml:"api_server_host,omitempty"`
-	UIServerHost       string              `json:"ui_server_host" yaml:"ui_server_host,omitempty"`
-	APIKey             string              `json:"api_key" yaml:"api_key,omitempty"`
-	User               string              `json:"user" yaml:"user,omitempty"`
-	UncommittedChanges bool                `json:"patch_uncommitted_changes" yaml:"patch_uncommitted_changes,omitempty"`
-	PreserveCommits    bool                `json:"preserve_commits" yaml:"preserve_commits,omitempty"`
-	Projects           []ClientProjectConf `json:"projects" yaml:"projects,omitempty"`
-	Admin              ClientAdminConf     `json:"admin" yaml:"admin,omitempty"`
-	LoadedFrom         string              `json:"-" yaml:"-"`
+	APIServerHost         string              `json:"api_server_host" yaml:"api_server_host,omitempty"`
+	UIServerHost          string              `json:"ui_server_host" yaml:"ui_server_host,omitempty"`
+	APIKey                string              `json:"api_key" yaml:"api_key,omitempty"`
+	User                  string              `json:"user" yaml:"user,omitempty"`
+	UncommittedChanges    bool                `json:"patch_uncommitted_changes" yaml:"patch_uncommitted_changes,omitempty"`
+	PreserveCommits       bool                `json:"preserve_commits" yaml:"preserve_commits,omitempty"`
+	Projects              []ClientProjectConf `json:"projects" yaml:"projects,omitempty"`
+	Admin                 ClientAdminConf     `json:"admin" yaml:"admin,omitempty"`
+	LoadedFrom            string              `json:"-" yaml:"-"`
+	DontSetDefaultProject bool                `json:"dont_set_default_project" yaml:"dont_set_default_project"`
```

Comment:
It's a bit odd to have a contraction as part of a parameter. "disable_auto_setting_defaults" maybe? or something like that

Reply:
yea I like that a lot better. Fixed